.travis.yml

```diff
@@ -1,4 +1,4 @@
-dist: bionic
+dist: focal
```

Comment:
Nice, didn't know they support focal yet.

Reply:
realized it just right now, the `libxen-dev` is too old for bionic and there are no headers for foreignmemory etc.

anyway with focal we get some random compilation errors :/
 
https://travis-ci.org/github/tklengyel/drakvuf/jobs/726996066

idk what shall I do with it now

Reply:
Get clang-10:
```
      addons:
        apt:
          sources:
          - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main'
            key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
          - ubuntu-toolchain-r-test
          packages:
          - clang-10

Reply:
I've added this snippet below, but the pipeline still fails with random errors

https://travis-ci.org/github/tklengyel/drakvuf/jobs/727036105

did I miss something else? It's even more confusing that I've tested a package build for ubuntu:focal and it works so it doesn't compile only on Travis o_O